Dockerfile.template

```diff
@@ -84,7 +86,7 @@ RUN set -ex; \
 	\
 # explicitly update autoconf config.guess and config.sub so they support more arches/libcs
 	for f in config.guess config.sub; do \
-		wget -O "$f" "https://git.savannah.gnu.org/cgit/config.git/plain/$f?id=7d3d27baf8107b630586c962c057e22149653deb"; \
+		curl -sL -o "$f" "https://git.savannah.gnu.org/cgit/config.git/plain/$f?id=7d3d27baf8107b630586c962c057e22149653deb"; \
```

Comment:
I'm not a fan of this -- `wget` is much simpler to get "correct" with validation behavior.  For example, this is missing `-f` on the `curl` invocation, so failures will be silently ignored.